README.md

```diff
@@ -24,16 +24,7 @@ We are in the process of adding Python 3 support.
 
 1. Create or activate a virtualenv for working on govdelivery.
 1. `pip install -e .`
-1. `cd govdelivery/`
-1. Export the following environment variables corresponding to your account on
-   the GovDelivery staging server:
-   - `GOVDELIVERY_USER`
-   - `GOVDELIVERY_PASSWORD`
-   - `GOVDELIVERY_ACCOUNT_CODE`
-   - `GOVDELIVERY_BASE_URL`
-1. `python -m unittest tests.test_api` and
-   `python -m unittest tests.test_xml_response_parsers`
-
+1. `python -m unittest govdelivery.tests`
```

Comment:
You remove the instructions to define these variables, but when I run the tests I get errors about a missing `GOVDELIVERY_USER` variable; see below.